react/country/SVK.js

```diff
@@ -0,0 +1,141 @@
+import { POSTAL_CODE } from '../constants'
+
+export default {
+  country: 'SVK',
+  abbr: 'SK',
+  postalCodeFrom: POSTAL_CODE,
+  postalCodeProtectedFields: ['state'],
+  fields: [
+    {
+      hidden: true,
+      name: 'country',
+      maxLength: 100,
+      label: 'country',
+      size: 'medium',
+    },
+    {
+      name: 'postalCode',
+      maxLength: 6,
+      label: 'postalCode',
+      required: true,
+      mask: '999 99',
+      regex: /^[0,8,9]\d{2}\ \d{2}$/, //shares zipcode range with Czech republic, 1-7 are reserved there.
```

Comment:
same as above, also you shouldn't include the commas inside the backets

```suggestion
      regex: /^[089]\d{2}\ \d{2}$/, //shares zipcode range with Czech republic, 1-7 are reserved there.
```